pandas/src/parser/tokenizer.c

```diff
@@ -743,26 +753,9 @@ int tokenize_delimited(parser_t *self, size_t line_limit)
 
         switch(self->state) {
 
-        case SKIP_LINE:
-            TRACE(("tokenize_delimited SKIP_LINE 0x%x, state %d\n", c, self->state));
-            if (c == '\n') {
-                END_LINE();
-            } else if (c == '\r') {
-                self->file_lines++;
-                self->state = EAT_CRNL_NOP;
-            }
-            break;
-
         case START_RECORD:
             // start of record
-            if (skip_this_line(self, self->file_lines)) {
-                self->state = SKIP_LINE;
-                if (c == '\n') {
-                    END_LINE();
-                }
-                break;
-            }
-            else if (c == '\n') {
+            if (c == '\n') {
```

Comment:
Because the `lineterminator` is not custom?  It never was `lineterminator` AFAICT, as you can see here <a href="https://github.com/pydata/pandas/pull/12900/files#diff-3010e5592cbe0505f6b76a93d861972eL765">here</a>.